### DISCUSSION TITLE
What do you use now instead of GetPixels()

### DISCUSSION THREAD

**Discussion by jimmywyatt** [dlemstra/Magick.NET#906] [Help] [answered]
I want to loop through all the pixels but GetPixels() seems to have been removed, how do you do it now?

**Comment by dlemstra** [comment, marked as answer]
I suspect you recently upgraded to a version that uses the "new" Core library. It's still there but in a different interface. You will need to use the `IMagickImage<QuantumType>` interface instead.

**Reply by jimmywyatt** [reply]
Thanks that's perfect, I saw this, but wasn't sure what the quantumtype meant in this context. Looking at it it seems I can just set it to a UInt16 for what I need.
